### ISSUE TITLE
Example is not accurate

### ISSUE THREAD

**Issue by peteraritchie** [CircleCI-Public/continuation-orb#18]
## Orb version:

0.2.0

## What happened:

using the example (updating version to latest orb version) from https://circleci.com/developer/orbs/orb/circleci/continuation:
```yaml
version: '2.1'
orbs:
  continuation: circleci/continuation@0.2.0
workflows:
  use-my-orb:
    jobs:
      - continuation/continue:
          configuration_path: ./config.yml
```
results in message: 

`CIRCLE_CONTINUATION_KEY is required. Make sure setup workflows are enabled.`

## Expected behavior:

no errors

Additional Information:

Example should include `setup: true`

**Comment by peteraritchie** [comment]
Does this Orb simply need to be deployed?  The current source seems correct.

**Comment by Markl121** [comment]
This issue has been resolved as of version 0.4.0
